hack/bazel-test.sh

```diff
@@ -5,4 +5,5 @@ source hack/config.sh
 
 bazel test \
     --config=${ARCHITECTURE} \
+    ${BAZEL_ARGS} \
```

Comment:
I like manipulating the args via `.bazelrc` files. The current `.bazelrc` file also check if a `user.bazelrc` and a `ci.bazelrc` file exists (both ignored via .gitignore). Would it make sense to, instead of this, check in `bazel-test.sh` if the jobs are run in CI and then simply dump a ci.bazelrc file?

```bash
if [ "${CI}" == "true" ]; then
cat >ci.bazelrc <<EOF
test --cache_test_results=no --runs_per_test=4 --test_timeout=30
EOF
fi
```

Locally someone can always write a `user.bazelrc` file with this content:

```
test --cache_test_results=no --runs_per_test=4 --test_timeout=30
```

Reply:
The main reason for me  to prefer .bazelrc files is, that one does not have to pass extra arguments to all kinds of places. Not so much an issue here, but if we run for instance multiple times `bazel test ...` for different parts, we have to ensure to always pass the extra args, vs. with .bazelrc files, we ensure that each `bazel test` invocation gets the right settings.

Reply:
Yeah the suggested approach looks much better. Previously I did not realize how `ci.bazelrc` was used. Lets give it a try...